schemas/groups/navigation.json

```diff
@@ -476,12 +476,22 @@
     },
     "headingMagnetic": {
       "$ref": "../definitions.json#/definitions/numberValue",
-      "description": "Current magnetic heading of the vessel",
+      "description": "Current magnetic heading of the vessel, equals 'headingRaw adjusted for magneticDeviation'",
+      "units": "rad"
+    },
+    "magneticDeviation": {
+      "$ref": "../definitions.json#/definitions/numberValue",
+      "description": "Magnetic deviation of the compass at the current headingRaw (if known)",
```

Comment:
I would leave ’if known’ out. It applies to a lot of items in the schema. Or is there a particular reason to add it here and not all over the place?